test/conf/autotest_okelmann_christina.cfg

```diff
@@ -0,0 +1,47 @@
+[local]
+moonprogs_dir = ${local:proot}/test/moonprogs/
+proot = /scratch/okelmann/vmuxIO
+# TODO make host driver configurable/use hosts.yml
+
+[host]
+fqdn=localhost
+admin_bridge=br-okelmann0
+admin_bridge_ip_net=192.168.56.1/24
+admin_tap=tap-okelmann0
+test_iface=enp24s0f1
+test_iface_addr=0000:18:00.1
+test_iface_mac=b4:96:91:aa:a1:c1
+test_iface_driv=ice
+test_bridge=br-okelmann1
+test_tap=tap-okelmann1
+test_macvtap=mvtap-okelmann1
+moongen_dir=${local:proot}/mgln/
+moonprogs_dir=${local:proot}/test/moonprogs/
+xdp_reflector_dir=${local:proot}/xdp/lib/
+qemu_path=${local:proot}/qemu-ioregionfd/bin/
+root_disk_file=${local:proot}/VMs/guest-image.qcow2
+ssh_config=${local:proot}/test/conf/ssh_config_localhost
+
+[guest]
+fqdn=guest.vmux.local
+vcpus=4
+memory=4096 # TODO isnt this hardcoded
```

Comment:
they dont seem to be used. They should be though. If time-consuming to implement, just remove here in the config files.

Reply:
Implemented in 994e8b4790e0a67ff8951a7bef4023d8a41803c6 ... See commit message for more details, tldr: autotest.cfg now sets the default vcpus and memory values ... run-guest command can overwrite via CLI options.